webapp/javascript/pages/TagExplorerView.tsx

```diff
@@ -224,6 +227,17 @@ function TagExplorerView() {
                 }
                 height="125px"
                 format="lines"
+                onHoverDisplayTooltip={(data) =>
+                  (
+                    <ExploreTooltip
+                      pageX={data.pageX}
+                      pageY={data.pageY}
+                      align={data.align}
+                      values={data.values}
+                      timeLabel={data.timeLabel}
+                    />
+                  ) as ShamefulAny
```

Comment:
I typed `ExploreTooltip` as `React.FC<ExploreTooltipProps>` everywhere, but for some reason it's "transformed" to Element. Maybe @eh-am you know the reason and how to fix it correctly?
![image](https://user-images.githubusercontent.com/7372044/188160020-cf9c8a31-b0ba-41d9-8ecc-4b5309e3ca42.png)

Reply:
what fields are missing?